packages/eslint/README.md

```diff
@@ -42,68 +42,53 @@ const eslint = require('@neutrinojs/eslint');
 
 // Usage shows default values
 neutrino.use(eslint, {
-  test: neutrino.regexFromExtensions(), // Uses extensions from neutrino.options.extensions
+  // Uses extensions from neutrino.options.extensions
+  test: neutrino.regexFromExtensions(),
   include: [neutrino.options.source, neutrino.options.tests],
   exclude: [],
   eslint: {
+    // For supported options, see:
+    // https://github.com/webpack-contrib/eslint-loader#options
+    // https://eslint.org/docs/developer-guide/nodejs-api#cliengine
     cache: true,
+    // Make errors fatal not just for 'production' but also 'test'.
     failOnError: process.env.NODE_ENV !== 'development',
     cwd: neutrino.options.root,
     useEslintrc: false,
-    root: true,
-    // Can be the name of a built-in ESLint formatter or the module/path of an external one.
+    // Can be the name of a built-in ESLint formatter
+    // or the module/path of an external one.
     formatter: 'codeframe',
-    plugins: ['babel'],
-    baseConfig: {},
-    envs: ['es6'],
-    parser: 'babel-eslint',
-    parserOptions: {
-      ecmaVersion: 2018,
-      sourceType: 'module'
-    },
-    settings: {},
-    globals: ['process'],
-    rules: {}
+    // The options under `baseConfig` correspond to those
+    // that can be used in an `.eslintrc.*` file.
+    baseConfig: {
+      env: {
+        es6: true
+      },
+      extends: [],
```

Comment:
I've deliberately included empty config options here to give hints (both in the docs and via `--inspect`) as to where the correct place is to pass each of the options.